### ISSUE TITLE
how to run it

### ISSUE THREAD

**Issue by ignat-admin** [esin/telegram_online#4]
I don't quite understand how to run this application,
can you explain in more detail?

**Comment by esin** [comment]
1. You should have Linux
2. You should install Docker
3. Clone telegram_online
4. Run ./run_docker.sh and enter necessary information
5. Press ctrl+s to stop container
6. Run again ./run_docker.sh :)
7. Done

**Comment by ignat-admin** [comment]
Will it be possible to run this on Windows?

**Comment by esin** [comment]
Well, really - don't know
Don't have any Windows :( 

I think, if you have Docker, you can try to open run_docker.sh with notepad and copy commands from it into command line and may it will work

May be(!) run_docker.sh will work with WSL (have never seen it, but hope dies last)

**Comment by ignat-admin** [comment]
I did, you need to create a .py file and place it in a shared folder, here are its contents:
import os

if not os.path.exists("session.madeline"):
    print("First run")
    os.system('docker run -ti -v %cd%:/app hub.madelineproto.xyz/danog/madelineproto:latest php /app/online.php')
else:
    os.system('docker run --restart always -d --name telegram_online -v %cd%:/app hub.madelineproto.xyz/danog/madelineproto:latest php /app/online.php')

**Comment by ignat-admin** [comment]
а можно эту прогу сделать без докера, он чет комп грузит

**Comment by esin** [comment]
> а можно эту прогу сделать без докера, он чет комп грузит

Да, конечно, но необходимо настроить окружение для запуска php скрипта

Готовый docker образ обеспечивает это окружение

**Comment by jooblex** [comment]
Блин, всё равно не понятно

**Comment by jooblex** [comment]
Можно полную инструкцию я огурчик

**Comment by esin** [comment]
> Можно полную инструкцию я огурчик
> 

Инструкция для огурчиков

Есть ли Linux и Docker?

Есть ли сервер, который запущен 24/7?
